src/Mentat.UI/Views/Mentor/SubmitForm.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿
+@{
+    ViewData["Title"] = "SubmitForm";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h1>SubmitForm</h1>
```

Comment:
Since its user facing put a space between words please.

Reply:
Should this actually say something like 'you've successfully submitted your form'? Or something like that, since it displays after submitting the form.